pkgs/development/libraries/science/math/nccl/default.nix

```diff
@@ -38,6 +38,10 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
+  passthru = {
+    inherit cudatoolkit;
+  };
```

Comment:
This is necessary for the `assert` in cupy/default.nix.

Reply:
👍🏻